velox/functions/prestosql/benchmarks/IntArrayProxyBenchmark.cpp

```diff
@@ -206,6 +206,25 @@ class ArrayProxyBenchmark : public functions::test::FunctionBenchmarkBase {
     folly::doNotOptimizeAway(cnt);
   }
 
+  size_t runStdRef() {
+    for (auto k = 0; k < 100; k++) {
+      std::vector<std::vector<std::optional<int>>> arrayVector;
+      for (auto i = 0; i < 1000; i++) {
```

Comment:
I wonder what the difference in time will be if you call reserve() for current here since you are aware of final array size.

Reply:
+1 think `resize(1000)` would be a more fair comparison.